CONTRIBUTING.md

```diff
@@ -37,11 +37,19 @@ npx vsce package -o vscode-neovim.vsix
 
 ### Logging
 
-You can observe the extension logs via the `vscode-neovim` Output channel or from the dev tools console (run the
-`Developer: Toggle Developer Tools` vscode command to see the console).
+You can view the extension logs in one of three locations
 
-Note: some messages are not logged to the Output channel, to avoid infinite loop. This is decided by the
-[`logToOutputChannel` parameter](https://github.com/vscode-neovim/vscode-neovim/blob/7337ffd5009067d074af5371171f277cb522aa9b/src/logger.ts#L184).
+1. Via the `vscode-neovim` Output channel
+
+    - **NOTE**: some messages are not logged to the Output channel, to avoid infinite loop. This is decided by the
+      [`logToOutputChannel` parameter](https://github.com/vscode-neovim/vscode-neovim/blob/7337ffd5009067d074af5371171f277cb522aa9b/src/logger.ts#L184).
+
+2. From the dev tools console (run the `Developer: Toggle Developer Tools` vscode command to see the console) by
+   enabling the `vscode-neovim.logOutputToConsole` setting.
+3. From a log file of your choosing, by configuring the `vscode-neovim.logPath` setting.
+
+VSCode, by default will only show messages at the "Info" level or above, but than can be changed by running the command
+"Developer Set Log Level..." -> "vscode-neovim" and selecting the desired log level.
```

Comment:
`Developer: Set Log Level...` is a drastic way to set the log level, because it increases the logs for all vscode components.

The log-level can also be set per-channel. We should definitely mention that, and also mention that `Set As Default` should be selected if you want it to persist.

<img width="335" alt="image" src="https://github.com/vscode-neovim/vscode-neovim/assets/1359421/dc843206-4ccf-4b7a-ba86-a3f39e059943">

```suggestion
`Developer: Set Log Level...` -> "vscode-neovim" and selecting the desired log level.
```

Reply:
> Given this, is there something different between what your screenshot does and what I have written here?

Oh my mistake. I see also that one can hover over the checkmark icon in that menu to `Set as default`:
<img width="771" alt="image" src="https://github.com/vscode-neovim/vscode-neovim/assets/1359421/71eaa30e-a754-4ca3-a884-5df653abc964">